README.adoc

```diff
@@ -289,15 +302,29 @@ InventoryConfig.java
 ----
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryConfig.java[]
 ----
+
 Inject the [hotspot=custom-converter file=3]`io_openliberty_guides_email` property, and add the [hotspot=getEmail file=3]`getEmail()` method.
 
+Repeat here.
```

Comment:
Should explicitly say that you repeat the same in the `SystemConfig` file instead of just saying "repeat here".